src/Client/Core/DurableTaskClient.cs

```diff
@@ -337,7 +337,7 @@ public virtual Task ResumeInstanceAsync(string instanceId, CancellationToken can
     public abstract AsyncPageable<OrchestrationMetadata> GetAllInstancesAsync(OrchestrationQuery? filter = null);
 
     /// <inheritdoc cref="PurgeInstanceAsync(string, PurgeInstanceOptions, CancellationToken)"/>
-    public virtual Task<PurgeResult> PurgeInstancesAsync(string instanceId, CancellationToken cancellation)
+    public virtual Task<PurgeResult> PurgeInstanceAsync(string instanceId, CancellationToken cancellation)
```

Comment:
The previous change was the breaking change, this fixes it.

Reply:
But since the typo was already released, I suppose this is, no matter what, a breaking change (as it was made public). But that's a nitpick, I think it makes sense to revert this typo.